### ISSUE TITLE
crch and cevt None in SSM

### ISSUE THREAD

**Issue by jtwhite79** [modflowpy/flopy#268]
If crch and/or cevt are None in the SSM, their entries are skipped when writing the SSM, which then results in a stdin prompt from MT3D to enter the unit number.  I think most of us prob want these to be zero by default to avoid this prompting.  Any reason not to make these default of zero instead of None?

**Comment by emorway-usgs** [comment]
cevt should default to 0 - it is hard to think of a dissolved constituent leaving with the ET rather than being left behind and evapo-concentrating.  As for crch, this is likely how much of the loading happens in non-point source systems, so the user will in all likelihood be specifying non-zero values for crch.  For point-source problems, a default of crch=0 makes sense for most of the model (everywhere except the point source).  crch and cevt both equal to zero would be a good way to go, in my opinion.

**Comment by langevin-usgs** [comment]
I agree with you, Jeremy.  If they are None, then the default concentration should be zero, and those zero value array headers needs to be written so the model doesn't prompt you for a file.

**Comment by jdhughes-usgs** [comment]
@langevin-usgs I suspect this one can be closed
